### ISSUE TITLE
Documentation mismatch

### ISSUE THREAD

**Issue by pbagchi** [linkerd/linkerd#2017]
Issue Type:

- [X] Bug report

Documentation mismatch for https://linkerd.io/config/1.4.2/linkerd/index.html#aperture-least-loaded.
Key smoothWin throws UnrecognizedPropertyException. Log message shows the right key.
```
com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "smoothWin" (class io.buoyant.linkerd.Aperture), not marked as ignorable (7 known properties: "maxEffort", "minAperture", "smoothWindowMs", "highLoad", "lowLoad", "enableProbation", "kind"])
```

**Comment by adleong** [comment]
Nice find, @pbagchi.  Are you interested in submitting a PR to correct this?

**Comment by pbagchi** [comment]
@adleong - Sure, I can do that.

**Comment by dadjeibaah** [comment]
Marking this closed based on #2043
